sdf.md

```diff
@@ -1124,17 +1122,16 @@ present specification.
 |---------------+---------------------------------------------+--------------------------------------------------------------------------+---------|
 | (common)      |                                             | {{common-qualities}}                                                     |         |
 | unit          | string                                      | unit name (note 1)                                                       | N/A     |
-| scaleMinimum  | number                                      | lower limit of value in units given by unit (note 2)                     | N/A     |
-| scaleMaximum  | number                                      | upper limit of value in units given by unit (note 2)                     | N/A     |
 | nullable      | boolean                                     | indicates a null value is available for this type                        | true    |
-| contentFormat | string                                      | content type (IANA media type string plus parameters), encoding (note 3) | N/A     |
+| contentFormat | string                                      | content type (IANA media type string plus parameters), encoding (note 2) | N/A     |
 | sdfType       | string ({{sdftype}})                        | sdfType enumeration (extensible)                                         | N/A     |
 | sdfChoice     | named set of data qualities ({{sdfchoice}}) | named alternatives                                                       | N/A     |
 | enum          | array of strings                            | abbreviation for string-valued named alternatives                        | N/A     |
 {: #sdfdataqual2 title="SDF-defined Qualities of sdfData"}
 
 
-1. Note that the quality `unit` was called `units` in SDF draft 1.0.
+1. Note that the quality `unit` was called `units` in earlier drafts
+   of SDF.
```

Comment:
Would “earlier drafts of this specification” work better than “drafts of SDF”?

Reply:
> Would “earlier drafts of this specification” work better than “drafts of SDF”?

Ja.  Fix s.u.